packages/editor/src/components/collab-sidebar/index.js

```diff
@@ -0,0 +1,198 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	comment as commentIcon,
+	commentAuthorAvatar as userIcon,
+	Icon,
+	check as resolvedIcon,
+} from '@wordpress/icons';
+import { useSelect } from '@wordpress/data';
+import {
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	__experimentalText as Text,
+} from '@wordpress/components';
+import { dateI18n, format, getSettings } from '@wordpress/date';
+
+/**
+ * Internal dependencies
+ */
+import PluginSidebar from '../plugin-sidebar';
+import { store as editorStore } from '../../store';
+
+const isBlockCommentExperimentEnabled =
+	window?.__experimentalEnableBlockComment;
+
+/**
+ * Renders the Collab sidebar.
+ */
+export default function CollabSidebar() {
+	const { threads } = useSelect( ( select ) => {
+		const meta = select( editorStore ).getEditedPostAttribute( 'meta' );
+		return {
+			threads: meta?.collab ? JSON.parse( meta.collab ) : false,
+		};
+	}, [] );
+
+	// Check if the experimental flag is enabled.
+	if ( ! isBlockCommentExperimentEnabled ) {
+		return null; // or maybe return some message indicating no threads are available.
+	}
+
+	// Get the date time format from WordPress settings.
+	const dateTimeFormat = getSettings().formats.datetime;
+
+	return (
+		<PluginSidebar
+			name="collab-activities"
+			title={ __( 'Collab Activities' ) }
+			icon={ commentIcon }
+		>
+			<div className="editor-collab-sidebar__activities">
+				{
+					// If there are no threads, show a message indicating no threads are available.
+					! threads && (
+						<VStack
+							className="editor-collab-sidebar__thread"
+							spacing="3"
+						>
+							<HStack
+								alignment="left"
+								spacing="3"
+								justify="flex-start"
+							>
+								{ __( 'No comments available' ) }
+							</HStack>
+						</VStack>
+					)
+				}
+				{ Object.keys( threads ).length > 0 &&
+					Object.values( threads )
+						.reverse()
```

Comment:
What does reverse do here? Should they be started by date, and grouped by block position?

Reply:
Reverse will display the latest activities at the top.